src/types/events/voice_gateway/speaking.rs

```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use bitflags::bitflags;
+use serde::{Deserialize, Serialize};
+
+use crate::types::{Snowflake, WebSocketEvent};
+
+/// Event that tells the server we are speaking;
+///
+/// Essentially, what allows us to send udp data and lights up the green circle around your avatar.
```

Comment:
<!--
 Copyright (c) 2024 star
 
 This Source Code Form is subject to the terms of the Mozilla Public
 License, v. 2.0. If a copy of the MPL was not distributed with this
 file, You can obtain one at https://mozilla.org/MPL/2.0/.
-->


```suggestion
/// Essentially, what allows us to send UDP data and lights up the green circle around your avatar.
```